admin/app/controllers/solidus_admin/adjustment_reasons_controller.rb

```diff
@@ -52,10 +54,10 @@ def create
     end
 
     def edit
-      set_index_page
-
       respond_to do |format|
-        format.html { render component('adjustment_reasons/edit').new(page: @page, adjustment_reason: @adjustment_reason) }
+        format.html do
+          render component('adjustment_reasons/edit').new(adjustment_reason: @adjustment_reason), layout: false
```

Comment:
Ideally we want to make the layout toggle by context

```suggestion
          render component('adjustment_reasons/edit').new(adjustment_reason: @adjustment_reason), layout: !turbo_frame_request?
```

Reply:
I added a general layout toggle for turbo frame requests in #6046 https://github.com/solidusio/solidus/pull/6046/commits/8fad74c4ab05dc38c06c428a8c4590524e648f8e